nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestValidateCsv.java

```diff
@@ -333,4 +333,23 @@ public void testMultipleRuns() {
         runner.assertTransferCount(ValidateCsv.REL_VALID, 2);
         runner.assertTransferCount(ValidateCsv.REL_INVALID, 0);
     }
+
+    @Test
+    public void testEscapingLineByLine() {
+        final TestRunner runner = TestRunners.newTestRunner(new ValidateCsv());
+        runner.setProperty(ValidateCsv.DELIMITER_CHARACTER, ",");
+        runner.setProperty(ValidateCsv.END_OF_LINE_CHARACTER, "\r\n");
+        runner.setProperty(ValidateCsv.QUOTE_CHARACTER, "\"");
+        runner.setProperty(ValidateCsv.HEADER, "false");
+        runner.setProperty(ValidateCsv.VALIDATION_STRATEGY, ValidateCsv.VALIDATE_LINES_INDIVIDUALLY);
+
+        runner.setProperty(ValidateCsv.SCHEMA, "ParseInt(),ParseInt(),ParseInt()");
+
+        runner.enqueue("Field1,\"Field2,excaped\",Field3");
```

Comment:
@pvillard31 Suggestion: extract into `final String` and use in an assertion? I.e.

```
final String payload = "Field1,\"Field2,escaped\",Field3"
runner.enqueue(payload)
...
runner.getFlowFilesForRelationship(ValidateCsv.REL_INVALID).get(0).assertContentEquals(payload)
```

Also, `excaped` is definitely a typo in my template, no need to repeat it here.

Reply:
@pvillard31 Also, I've observed roughly the same behavior with "valid" output, but couldn't distill it to a minimal example. Maybe worth adding/extending a test for "valid" as well?